conf/application.conf

```diff
@@ -81,55 +74,12 @@ application.langs="en"
 # !!!WARNING!!! DO NOT CHANGE THIS ROUTER
 application.router=prod.Routes
 
-
-# Controller
-# ~~~~~
-# By default all controllers will have authorisation, logging and
-# auditing (transaction monitoring) enabled.
-# The below controllers are the default exceptions to this rule.
-
-controllers {
-    # 300 is the default, you may need to change this according to your needs
-    confidenceLevel = 300
-
-    com.kenshoo.play.metrics.MetricsController = {
```

Comment:
I'm guessing you've removed this because it's not needed. Is it 100% not needed anymore?

Reply:
The `com.kenshoo.play.metrics.MetricsController` config is already defined via the include of "backend.conf" from bootstrap-play.  Same for the `metrics` config - we just need to override 2 of the "backend.conf" values to retain what we had before.

The `logger` settings, if not removed, simply generate a warning that they are being ignored and that you should be using a logback configuration file instead.

The `confidenceLevel` configuration is the most contentious.  It seems to relate to [play-authorisation](https://github.com/hmrc/play-authorisation), but as far as I can tell this service implements its own authorisation strategy and is not using play-authorisation.